icons/githubactions/githubactions-plain.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
same as with githubactions-plain.svg, this needs to be minimized :) See the comment on the other icon for info on how to do that